File: ganapathy_pavers/ganapathy_pavers/report/monthly_fencing_post_production_report/monthly_fencing_post_production_report.py
# Copyright (c) 2022, Thirvusoft and contributors
# For license information, please see license.txt

from frappe import _
from ganapathy_pavers.ganapathy_pavers.report.monthly_compound_wall_production_report.monthly_compound_wall_production_report import execute as compound_wall_report

def execute(filters=None):
	return compound_wall_report(filters, ["Fencing Post"], "fencing_post")

File: ganapathy_pavers/ganapathy_pavers/report/monthly_fencing_post_production_report/monthly_fencing_post_production_report.js
// Copyright (c) 2022, Thirvusoft and contributors
// For license information, please see license.txt
/* eslint-disable */

frappe.query_reports["Monthly Fencing Post Production Report"] = {
	"filters": [
		{
			"fieldname":"from_date",
			"label": __("From Date"),
			"fieldtype": "Date",
			"width": "80",
			"default": frappe.datetime.month_start(),
			"reqd":1
		},
		{
			"fieldname":"to_date",
			"label": __("To Date"),
			"fieldtype": "Date",
			"default": frappe.datetime.month_end(),
			"width": "80",
			"reqd":1
		},
		{
			"fieldname": "expense_summary",
			"label": __("Expense Summary"),
			"fieldtype": "Check",
		},
		{
			"fieldname": "new_method",
			"label": __("New Expense Method"),
			"fieldtype": "Check",
			"default": 0,
		}
	]
};
